docs/en/sql-reference/functions/introspection.md

```diff
@@ -113,6 +113,111 @@ trace_source_code_lines: /lib/x86_64-linux-gnu/libpthread-2.27.so
 /build/glibc-OTsEL5/glibc-2.27/misc/../sysdeps/unix/sysv/linux/x86_64/clone.S:97
 ```
 
+## addressToLineWithInlines {#addresstolinewithinlines}
+
+Similar to `addressToLine`, but it will return an Array with all inline functions.
```

Comment:
Also please add a note that `addressToLineWithInlines` may be slower than `addressToLine`.